packages/dynamite/dynamite/lib/src/builder/generate_schemas.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:code_builder/code_builder.dart';
+import 'package:dynamite/src/builder/resolve_type.dart';
+import 'package:dynamite/src/builder/state.dart';
+import 'package:dynamite/src/helpers/dart_helpers.dart';
+import 'package:dynamite/src/models/openapi.dart' as openapi;
+import 'package:dynamite/src/models/type_result.dart';
+
+Iterable<Spec> generateSchemas(
+  final openapi.OpenAPI spec,
+  final State state,
+) sync* {
+  if (spec.components?.schemas != null) {
+    for (final schema in spec.components!.schemas!.entries) {
+      final identifier = toDartName(schema.key, uppercaseFirstCharacter: true);
+      if (schema.value.type == null && schema.value.ref == null && schema.value.ofs == null) {
+        yield TypeDef(
```

Comment:
@provokateurin can you please provide a spec that hits those two `TypeDef`s?
I don't entirely understand what they try to accomplish and they aren't covered by any of the nc specs.

As mentioned above these are also broken without this PR.

Reply:
I don't know where these originate from and I think I never had them used. They also don't make any sense imo, so just remove them